src/main/java/org/broadinstitute/hellbender/utils/config/GATKConfig.java

```diff
@@ -0,0 +1,135 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.Accessible;
+import org.aeonbits.owner.Mutable;
+import org.aeonbits.owner.Config.LoadPolicy;
+import org.aeonbits.owner.Config.LoadType;
+import org.aeonbits.owner.Config.Sources;
+
+import java.util.List;
+
+/**
+ * Configuration file for GATK options.
+ */
+@LoadPolicy(LoadType.MERGE)
+@Sources({
+        "file:${" + GATKConfig.CONFIG_FILE_VARIABLE_NAME + "}",                         // Variable for file loading
+        "file:GATKConfig.properties",                                                   // Default path
+        "classpath:org/broadinstitute/hellbender/utils/config/GATKConfig.properties"    // Class path
+})
+public interface GATKConfig extends Mutable, Accessible {
+
+    // =================================================================================================================
+    // =================================================================================================================
+    // Meta Options:
+    // =================================================================================================================
+    // =================================================================================================================
+
+    String CONFIG_FILE_VARIABLE_NAME = "pathToGatkConfig";
+
+    // =================================================================================================================
+    // =================================================================================================================
+    // System Options:
+    // =================================================================================================================
+    // =================================================================================================================
+
+    String SYSTEM_PROPERTY_PREFIX = "system.";
+
+    // ----------------------------------------------------------
+    // General Options:
+    // ----------------------------------------------------------
+
+    @Key("system.gatk_stacktrace_on_user_exception")
+    @DefaultValue("true")
+    boolean gatk_stacktrace_on_user_exception();
```

Comment:
This comes back to a question I had about the configuration fields / parameters.

Owner lets you look up values using the method names by default or by using the `Key` annotation.  We can make all the lookup methods standard camel case, but then we'd need to either make the properties camel case or add in the `Key` annotation to them.

More generally we need to standardize the way we want to name our configuration properties.  I'm fine with the camel case lookup, as long as we make everything consistent.  @droazen what do you think?